md_api/pom.xml

```diff
@@ -27,6 +27,19 @@
 
     <dependencies>
 
+
+        <dependency>
+            <groupId>org.codehaus.jackson</groupId>
+            <artifactId>jackson-mapper-asl</artifactId>
+            <version>1.9.13</version>
```

Comment:
add versions in settings.xml, provide variable name here.. see other dependencies for example